package.json

```diff
@@ -27,52 +27,45 @@
     ]
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^3.14.1",
-    "@semantic-release/changelog": "^6.0.1",
-    "@semantic-release/commit-analyzer": "^9.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@semantic-release/github": "^8.0.5",
-    "@semantic-release/npm": "^9.0.1",
-    "@semantic-release/release-notes-generator": "^10.0.3",
-    "@types/jest": "^27.5.2",
+    "@cloudflare/workers-types": "^3.11.0",
+    "@types/jest": "^29.0.0",
     "@types/service-worker-mock": "^2.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "eslint": "^7.32.0",
+    "@typescript-eslint/eslint-plugin": "^5.36.2",
+    "@typescript-eslint/parser": "^5.36.2",
+    "eslint": "^8.23.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-typescript": "^3.0.0",
-    "jest": "^27.5.1",
-    "miniflare": "^2.7.1",
-    "prettier": "^2.7.1",
-    "semantic-release": "^19.0.5",
+    "jest": "^29.0.2",
+    "miniflare": "^2.4.0",
+    "prettier": "^2.3.0",
     "service-worker-mock": "^2.0.5",
-    "terser-webpack-plugin": "^5.3.5",
```

Comment:
@jay-dee7 Why are we accepting older versions?

Reply:
this is accidental, I missed it when doing a rebase. I'll fix it right away